src/hotspot/share/compiler/compilerDefinitions.cpp

```diff
@@ -479,23 +479,23 @@ bool CompilerConfig::check_args_consistency(bool status) {
   uint min_code_cache_size = CodeCacheMinimumUseSpace DEBUG_ONLY(* 3);
   if (ReservedCodeCacheSize < InitialCodeCacheSize) {
     jio_fprintf(defaultStream::error_stream(),
-                "Invalid ReservedCodeCacheSize: %dK. Must be at least InitialCodeCacheSize=%dK.\n",
+                "Invalid ReservedCodeCacheSize: " UINTX_FORMAT "K. Must be at least InitialCodeCacheSize=" UINTX_FORMAT "K.\n",
```

Comment:
It seems like all these size values ought to be `size_t` rather than `uintx`, in which case could use "%zu".
I don't know what the fannout for changing the types might be.

Reply:
Fixed.